api/app/tests/fire_behavior/test_wind_speed.py

```diff
@@ -23,3 +23,48 @@ def test_wind_speed_result_no_wind_speed():
     assert math.isclose(wind_speed_result.wind_speed, 1.0, abs_tol=0.001)
     assert math.isclose(wind_speed_result.fwi, 0.0, abs_tol=0.001)
     assert wind_speed_result.status == 'ACTUAL'
+
+def test_wind_speed_result_with_wind_speed():
+    wind_speed_result = calculate_wind_speed_result(
+        requested_station=StationRequest(station_code=1, fuel_type=FuelTypeEnum.C2, wind_speed=5),
+        yesterday={"fineFuelMoistureCode": 1},
+        raw_daily={
+            "buildUpIndex": 1,
+            "duffMoistureCode": 1,
+            "droughtCode": 1,
+            "temperature": 1,
+            "relativeHumidity": 1,
+            "precipitation": 1,
+            "windSpeed": 1,
+            "recordType": {"id": "ACTUAL"},
+        },
+    )
+    assert math.isclose(wind_speed_result.bui, 0.606, abs_tol=0.001)
+    assert math.isclose(wind_speed_result.ffmc, 31.427, abs_tol=0.001)
+    assert math.isclose(wind_speed_result.isi, 0.006, abs_tol=0.001)
+    assert math.isclose(wind_speed_result.wind_speed, 5.0, abs_tol=0.001)
+    assert math.isclose(wind_speed_result.fwi, 0.001, abs_tol=0.001)
+    assert wind_speed_result.status == "ADJUSTED"
+
+
+def test_wind_speed_result_with_precipitation():
```

Comment:
Do we want to assert that the requested precipitation is set in the test?

Reply:
I've renamed `WindResult` to `AdjsutedFWIResult` and renamed some files. I now return the precip value as part of `AdjustedFWIResult` and have included a test.
Done in https://github.com/bcgov/wps/pull/4407/commits/149b8a50e8291b98e6a12fe9f57677c64a451065